pizza_cutter/des_pizza_cutter/_coadd_slices.py

```diff
@@ -75,44 +75,67 @@ def _build_slice_inputs(
+
     reject_outliers : bool
-        See the documentation of `make_des_pizza_slices` in
-        `pizza_cutter.des_pizz_cutter`.
+        If True, assume the SE images are approximatly registered with
+        respect to one another and apply the pixel outlier rejection
+        code from the `meds` package. If False, this step is skipped.
     symmetrize_masking : bool
-        See the documentation of `make_des_pizza_slices` in
-        `pizza_cutter.des_pizz_cutter`.
-    coadding_weight : str
-        See the documentation of `make_des_pizza_slices` in
-        `pizza_cutter.des_pizz_cutter`.
+        If True, the bit masks and any zero weight pixels will be rotated
+        by 90 degrees and applied again to the weight and bit masks. If False,
+        this step is skipped.
     noise_interp_flags : int
-        See the documentation of `make_des_pizza_slices` in
-        `pizza_cutter.des_pizz_cutter`.
-    se_interp_flags : int
-        See the documentation of `make_des_pizza_slices` in
-        `pizza_cutter.des_pizz_cutter`.
+        An "or" of bit flags. Any pixel in the image with one or more of these
+        flags will be replaced by noise (computed from the weight map) before
+        coadding. This step is done after any mask symmetrization.
+    spline_interp_flags : int
+        An "or" of bit flags. Any pixel in the image with one or more of these
+        flags will be interpolated using a cubic order interpolant over the
+        good pixels. This step is done after symmetrization of the mask and
+        any noise interpolation via `noise_interp_flags`.
     bad_image_flags : int
-        See the documentation of `make_des_pizza_slices` in
-        `pizza_cutter.des_pizz_cutter`.
+        An "or" of bit flags. Any image the set of SE images with any pixel
+        in the coadding region set to one of these flags is ignored during
+        coadding.
     max_masked_fraction : float
-        See the documentation of `make_des_pizza_slices` in
-        `pizza_cutter.des_pizz_cutter`.
+        The maximum masked fraction an SE image can have before it is
+        excluded from the coadd. This masked fraction is computed from any
+        zero weight pixels, any picels with any of the se_interp_flags or
+        any pixels with any of the noise_interp_flags. It is the fraction of
+        the subset of the SE image that approximatly overlaps the final coadded
+        region.
     max_unmasked_trail_fraction : float
-        See the documentation of `make_des_pizza_slices` in
-        `pizza_cutter.des_pizz_cutter`.
+        The maximum unmasked bleed trail fraction an SE image can have
+        before it is exlcuded from the coadd. This parameter is the
+        fraction of the subset of the SE image that overlaps the coadd. See
+        the function `compute_unmasked_trail_fraction` in
+        `pizza_cutter.des_pizz_cutter._slice_flagging.` for details.
     mask_tape_bumps: boold
```

Comment:
may as well fix this typo while we are at it

Reply:
thanks for fixing!